### PR TITLE
Draft Akka gRPC 1.1.0

### PR DESCRIPTION
**Pull request by ignasi35**
References https://github.com/akka/akka-grpc/issues/1129

### CODE REVIEW

File: blog/_posts/2021-02-08-akka-grpc-1.1.0-released.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+### Supported by Lightbend
+
+This module is now officially
+[Supported](https://developer.lightbend.com/docs/introduction/getting-help/support-terminology.html)
+as part of a [Lightbend Subscription](https://www.lightbend.com/lightbend-subscription).
+
+
+## Changes since 1.0.x
+
+Akka gRPC 1.1.0 requires Akka 2.6.x and Akka HTTP 10.2.x. The upgrade should be seamless except when upstream projects introduced changes (e.g. new Akka HTTP API).
```
Comment (raboof) [inline blog/_posts/2021-02-08-akka-grpc-1.1.0-released.md:30]:
```suggestion
Akka gRPC 1.1.0 requires Akka 2.6.x and Akka HTTP 10.2.x. The upgrade should be seamless except when upstream projects introduced changes (e.g. new Akka HTTP API).
```
It should be seamless as long as you avoided deprecated API's before upgrading, shouldn't it?

File: blog/_posts/2021-02-08-akka-grpc-1.1.0-released.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+This release introduces a new, experimental client backend powered by Akka-HTTP. The default backend is still the classic client implemented using `grpc-java` which is based on Netty. You can enable the new backend using `akka.grpc.client."*".backend="akka-http"`. The Akka-HTTP backend for gRPC is built using the Akka HTTP client support for HTTP/2 introduced in [Akka HTTP 10.2.3](https://akka.io/blog/news/2021/01/18/akka-http-10.2.3-released).
```
Comment (raboof) [inline blog/_posts/2021-02-08-akka-grpc-1.1.0-released.md:32]: Maybe make it a little clearer that this client is really in the experimental/preview phase, and we still recommend the classic client for general use?

File: blog/_posts/2021-02-08-akka-grpc-1.1.0-released.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+For a full list of changes see the [releases overview](https://github.com/akka/akka-grpc/releases)
+
+## Credits
+
+For this release we had the help of 8 contributors – thank you all very much!
+
+```
+Commits Added Removed Author
+   32      93      96 Scala Steward
```
Comment (raboof) [inline blog/_posts/2021-02-08-akka-grpc-1.1.0-released.md:42]: I think we typically remove Scala Steward from the credits. (also LOL that it's the top committer)

**Review by raboof — APPROVED** [review]

File: blog/_posts/2021-02-08-akka-grpc-1.1.0-released.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+This module is now officially
+[Supported](https://developer.lightbend.com/docs/introduction/getting-help/support-terminology.html)
+as part of a [Lightbend Subscription](https://www.lightbend.com/lightbend-subscription).
+
+
+## Changes since 1.0.x
+
+Akka gRPC 1.1.0 requires Akka 2.6.x and Akka HTTP 10.2.x.
+
+This release introduces a new, experimental client backend powered by Akka-HTTP. The default backend is still the classic client implemented using `grpc-java` which is based on Netty. You can enable the new backend using `akka.grpc.client."*".backend="akka-http"`. The Akka-HTTP backend for gRPC is built using the Akka HTTP client support for HTTP/2 introduced in [Akka HTTP 10.2.3](https://akka.io/blog/news/2021/01/18/akka-http-10.2.3-released). The Akka-HTTP backend for Akka gRPC client is released as a preview and the recommendation for genreal porpose usage is still the existing netty based backend.
```
Comment (johanandren) [inline blog/_posts/2021-02-08-akka-grpc-1.1.0-released.md:32]: general

**Review by johanandren — APPROVED** [review]
LGTM except the one spelling error

File: blog/_posts/2021-02-08-akka-grpc-1.1.0-released.md (hunk 4 above)
Comment (ignasi35) [reply blog/_posts/2021-02-08-akka-grpc-1.1.0-released.md:32]: and `porpose` 🤦🏼‍♂️

Comment (ignasi35) [reply blog/_posts/2021-02-08-akka-grpc-1.1.0-released.md:32]:
```suggestion
This release introduces a new, experimental client backend powered by Akka-HTTP. The default backend is still the classic client implemented using `grpc-java` which is based on Netty. You can enable the new backend using `akka.grpc.client."*".backend="akka-http"`. The Akka-HTTP backend for gRPC is built using the Akka HTTP client support for HTTP/2 introduced in [Akka HTTP 10.2.3](https://akka.io/blog/news/2021/01/18/akka-http-10.2.3-released). The Akka-HTTP backend for Akka gRPC client is released as a preview and the recommendation for general purpose usage is still the existing, netty-based backend.
```

**Pull request merged**